jscript/jquery.js

```diff
@@ -0,0 +1,40 @@
+$(document).ready(function() {
+
+	$('.read-more').click(showMore);
+	$('.learn-more').click(learnMore);
+
+
+	function showMore () {
+		$('#readMoreText').slideDown(1000);
```

Comment:
Jut so you know, the numeric arguments you're passing into `slideDown()`, `hide()`, etc. are optional.  These "methods" internally have default values. By passing in numeric values, you're effectively saying to jQuery, "don't use your default values, use these values instead." This is of course completely allowed. I just wanted to make sure you're aware that you're not _required_ to pass in these arguments.